test/recipes/80-test_ssl_old.t

```diff
@@ -422,14 +422,12 @@ sub testssl {
         push(@protocols, "TLSv1.2") unless $no_tls1_2;
         push(@protocols, "SSLv3") unless $no_ssl3;
 	my $protocolciphersuitcount = 0;
-	my %ciphersuites =
-	    map { my @c =
-		      map { split(/:/, $_) }
-		      run(app(["openssl", "ciphers", "${_}:$ciphers"]),
-                          capture => 1);
-		  map { s/\R//; } @c;  # chomp @c;
-		  $protocolciphersuitcount += scalar @c;
-		  $_ => [ @c ] } @protocols;
+	my %ciphersuites = ();
+	foreach (map { s|\R||; $_ }
+		 run(app(["openssl", "ciphers", "-v"]), capture => 1)) {
```

Comment:
With the flag `capture => 1`, `run` returns the command output, not the exit code. 

I *could* change `run` to return `( $exit, @lines) `. We rarely use it for capturing, so the impact won't be big...